pkg/workload/tpcc/tpcc.go

```diff
@@ -273,6 +273,7 @@ var tpccMeta = workload.Meta{
 			`query-trace-file`:         {RuntimeOnly: true},
 			`fake-time`:                {RuntimeOnly: true},
 			`txn-preamble-file`:        {RuntimeOnly: true},
+			`repair-order-ids`:         {RuntimeOnly: true},
```

Comment:
There's an `order-id-repairs` above this, are these related/is one a typo?

Reply:
oh, you're totally right. nice catch. fixed.